engines/pyfunc-ensembler-service/Dockerfile

```diff
@@ -17,4 +17,4 @@ RUN conda env create -f ./env-${PYTHON_VERSION}.yaml -n $CONDA_ENV_NAME &&  \
     rm -rf /root/.cache
 
 # Install conda-pack:
-RUN conda install -c conda-forge conda-pack
\ No newline at end of file
+RUN conda install conda-pack
```

Comment:
`-c conda-forge` doesn't need to be specified explicitly anymore as it's the default channel.